global_variable_generator.py

```diff
@@ -85,7 +89,28 @@ def main():
             shelf_curr_addr += int(shelf_data['addr_offset'])
 
             write_rec_glob_var_table(global_var_table, name, addr, shelf_data['type'], shelf_data['init_value'])
+    
+    # parse constants and write into hmi_tag_table

```

Comment:
Seems like the array type is not handled in constant_tab.

Reply:
done